postgresql-plugin/src/main/java/io/cdap/plugin/postgres/PostgresDBRecord.java

```diff
@@ -85,7 +89,7 @@ private Object createPGobject(String type, String value, ClassLoader classLoader
   @Override
   protected void writeToDB(PreparedStatement stmt, Schema.Field field, int fieldIndex) throws SQLException {
     int sqlIndex = fieldIndex + 1;
-    ColumnType columnType = columnTypes.get(fieldIndex);
+    ColumnType columnType = modifiableColumnTypes.get(fieldIndex);
     if (PostgresSchemaReader.STRING_MAPPED_POSTGRES_TYPES_NAMES.contains(columnType.getTypeName()) ||
       PostgresSchemaReader.STRING_MAPPED_POSTGRES_TYPES.contains(columnType.getType())) {
       stmt.setObject(sqlIndex, createPGobject(columnType.getTypeName(),
```

Comment:
what is the use of createPGobject and can it affect our implementation if any of the column falls under this category.

Reply:
It will not have any impact on our implementation.

Reply:
have you tested with such type of column in postgres, if not please do it

Reply:
tested and working fine.